QUICKSTART.md

```
@@ -0,0 +1,398 @@
+**You should see:**
+```
+✓ Ready in 2.5s
+○ Compiling / ...
+✓ Compiled in 3s
+```
+
+Keep this terminal running! ✅
+
+### 6. Verify Setup
+
+Open your browser to **http://localhost:3000**
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)